ReactiveCocoa/Swift/Signal.swift

```diff
@@ -181,14 +181,24 @@ extension SignalType {
 		return observe(Observer(action))
 	}
 
-	/// Observes the Signal by invoking the given callback when `next` events are
-	/// received.
+	@available(*, unavailable, message="This Signal may emit errors which must be handled explicitly, or observed using observeResult:")
```

Comment:
The issue is that it breaks semantic versioning. That's liable to cause issues for people.

Calling it a bug fix is a bit of a stretch. ☺️ I agree that the method is unsafe, but we've let people use it, so we can't take it away without deprecating it first.